badge-maker/lib/badge-renderers.js

```diff
@@ -576,78 +576,103 @@ function forTheBadge({
   links,
   logo,
   logoWidth,
-  logoPadding,
```

Comment:
`logoPadding` is not a user-specified parameter, but a value computed here:

https://github.com/badges/shields/blob/f1a5dd9427a5eb9581fb7f0b82f5babe449ca1d6/badge-maker/lib/make-badge.js#L61

I think we probably ought to push this logic into the badge generation code as I've done here in for-the-badge.